apps/src/music/views/small-footer-music-overrides.scss

```diff
@@ -0,0 +1,17 @@
+@import 'color.scss';
```

Comment:
Should this just be part of the existing dark mode small footer?

Reply:
Music Lab's footer still has some slight differences with CodeBridge's, including a narrower width and a different background colour for the copyright button, so I've kept these additional overrides separate for now.  But as noted, would definitely like to refactor the small footer so these variants are done more neatly.

Reply:
Music Lab's footer still has some slight differences with CodeBridge's, including a narrower width and a different background colour for the two elements, so I've kept these additional overrides separate for now. But as noted, would definitely like to refactor the small footer so these variants are done more neatly.